test/globalconstraintregistrar.js

```diff
@@ -68,6 +68,19 @@ contract('GlobalConstraintRegistrar', function(accounts) {
      assert.equal(parameters[1],params.votingMachine.absoluteVote.address);
      });
 
+    it("proposeGlobalConstraint organizationsProposals", async function() {
+      var testSetup = await setup(accounts);
+      var globalConstraintMock = await GlobalConstraintMock.new();
+
+      var tx = await testSetup.globalConstraintRegistrar.proposeGlobalConstraint(testSetup.org.avatar.address,
+                                                                     globalConstraintMock.address,
+                                                                     "0x1234",
+                                                                     "0x1234");
+      var proposalId = await helpers.getValueFromLogs(tx, '_proposalId',1);
+      var organizationsProposals = await testSetup.globalConstraintRegistrar.organizationsProposals(testSetup.org.avatar.address,proposalId);
```

Comment:
should be  var organizationProposal

Reply:
Why is that since the contract property name is `organizationsProposals` ?

Reply:
Do you want the contract mapping name to be everywhere changed to `organizationProposals`?

Reply:
because this var hold a single proposal from a single avatar.

Reply:
-the var in the test hold a single proposal from a single avatar -so it should be name accordingly. 
-the mapping in the contract is a map between organization**s**  to their proposal**s**  so it should be name accordingly.